packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx

```diff
@@ -130,6 +132,28 @@ const TeamPromptOptionsMenu = (props: Props) => {
           openRecurrenceSettingsModal()
         }}
       />
+      <MenuItem
+        key='slack'
+        label={
+          <Link
+            to={`/team/${team.id}/settings/integrations`}
+            target='_blank'
+            rel='noopener noreferrer'
```

Comment:
+1 do we actually want to use a new tab? it's within our app, imo it'd be better to just use a normal link

Reply:
The thinking here is that even though it's within the app, it's still outside of the meeting, so I'm wary of taking the user completely out of the meeting potentially unexpectedly.

Reply:
🤔 If the user has added content but not clicked Submit, will they lose their work if they are taken to the Slack config page?